tbcontrol/symbolic.py

```diff
@@ -1,7 +1,7 @@
 """Control functions which operate symbolically using sympy"""
 
 import sympy
-
+import numpy
```

Comment:
I would prefer to keep these symbolic functions sympy-only instead of mixing in numpy.

Reply:
Sure.

Will change this